### PR TITLE
Ensure rubric view is accessible

### PR DESCRIPTION
**Pull request by shichongrui**
refs: MBL-12303
affects: Student
releast note: none

### CODE REVIEW

File: Student/Student/Submissions/SubmissionRubric/RubricCircleView.swift [hunk 1]
```diff
@@ -21,12 +21,11 @@ class RubricCircleView: UIView {
     private static let w: CGFloat = 49
     private static let space: CGFloat = 10
     private var buttons: [UIButton] = []
-    var rubric: RubricViewModel? {
```
Comment (shichongrui) [inline Student/Student/Submissions/SubmissionRubric/RubricCircleView.swift]: The `frame` was not correct when this was being run and thus the circles were being rendered off the screen. So doing it in `layoutSubviews` ensures the frame will be accurate when `setupButtons` is called.

File: Student/Student/Submissions/SubmissionRubric/RubricCircleView.swift [hunk 2]
```diff
@@ -64,11 +65,15 @@ class RubricCircleView: UIView {
             let font: UIFont
             let color: UIColor
             let bgColor: UIColor
+            let format = NSLocalizedString("g_points", bundle: .core, comment: "")
+            var a11yLabel: String = String.localizedStringWithFormat(format, Int(r))
+            a11yLabel += description
```
Comment (shichongrui) [inline Student/Student/Submissions/SubmissionRubric/RubricCircleView.swift]: I did all these (including line 76) separately as they each represent a piece of information I want VO to read and they aren't necessarily related.

File: Student/Student/Submissions/SubmissionRubric/RubricCircleView.swift [hunk 3]
```diff
@@ -78,11 +83,13 @@ class RubricCircleView: UIView {
             addSubview(button)
             buttons.append(button)
 
+            button.isUserInteractionEnabled = false
```
Comment (shichongrui) [inline Student/Student/Submissions/SubmissionRubric/RubricCircleView.swift]: I decided to leave the buttons in and just disable them for now so VO would inform the user the button is disabled.

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Please add a release note. If no release note is wanted, use `none`. Example: `release note: Fixed a bug that prevented users from enjoying the app.`
  </td>
    </tr>
  </tbody>
</table>



#### Affected Apps: Student

[MBL-12303](https://instructure.atlassian.net/browse/MBL-12303)

Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Core** | 93.52% | 93.5% | 0.01%
**Student** | 92.41% | 92.33% | 0.08%
**Teacher** | 100% | 100% | 0%
**React Native** | 92.72% | 92.72% | 0%
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: Core/Core/en.lproj/Localizable.stringsdict [hunk 4]
```diff
@@ -2,153 +2,169 @@
...
+			<key>two</key>
+			<string></string>
+			<key>few</key>
+			<string></string>
+			<key>many</key>
+			<string></string>
+			<key>other</key>
+			<string>%d Questions</string>
+		</dict>
+	</dict>
```
Comment (vanwagonet) [inline Core/Core/en.lproj/Localizable.stringsdict]: stupid Xcode re-indent. You don't need to fix this, but at some point we need to figure out how to get Xcode to not do this.

File: Core/Core/en.lproj/Localizable.stringsdict [hunk 5]
```diff
@@ -2,153 +2,169 @@
...
+			<key>many</key>
+			<string></string>
+			<key>other</key>
+			<string>%g pts</string>
+		</dict>
+	</dict>
+	<key>g_out_of_g_points_possible</key>
+	<dict>
+		<key>NSStringLocalizedFormatKey</key>
+		<string>Scored %g out of %#@points@ possible</string>
```
Comment (vanwagonet) [inline Core/Core/en.lproj/Localizable.stringsdict]: This was the actual change, right?

File: Student/Student/Submissions/SubmissionRubric/RubricCircleView.swift [hunk 6]
```diff
@@ -64,11 +65,15 @@ class RubricCircleView: UIView {
...
             let bgColor: UIColor
+            let format = NSLocalizedString("g_points", bundle: .core, comment: "")
+            var a11yLabel: String = String.localizedStringWithFormat(format, Int(r))
+            a11yLabel += description
 
             if selected {
                 font = UIFont.scaledNamedFont(.semibold20)
                 color = UIColor.white
                 bgColor = Brand.shared.primary
+                a11yLabel += NSLocalizedString("Selected", bundle: .core, comment: "")
```
Comment (vanwagonet) [inline Student/Student/Submissions/SubmissionRubric/RubricCircleView.swift]: Would `button.selected = true` give you the same or better result in VO?

File: Core/Core/en.lproj/Localizable.stringsdict (hunk 5 above)
Comment (shichongrui) [reply Core/Core/en.lproj/Localizable.stringsdict]: One of them, this change along with the the a11y label on each of the circle views.

File: Student/Student/Submissions/SubmissionRubric/RubricCircleView.swift (hunk 6 above)
Comment (shichongrui) [reply Student/Student/Submissions/SubmissionRubric/RubricCircleView.swift]: Oh good suggestion, I will give it a try.

File: Core/Core/en.lproj/Localizable.stringsdict (hunk 5 above)
Comment (vanwagonet) [reply Core/Core/en.lproj/Localizable.stringsdict]: I just meant in this file.

File: Student/Student/Submissions/SubmissionRubric/RubricCircleView.swift (hunk 6 above)
Comment (shichongrui) [reply Student/Student/Submissions/SubmissionRubric/RubricCircleView.swift]: Yeah I like this better.

**Comment by shichongrui** [comment]
@vanwagonet ready for another review.

**Pull request merged**